src/util/ValidationHelper.js

```diff
@@ -110,6 +111,7 @@ export function filterUserDataMin(userObject, types = []) {
     isCivicLikerTrial,
     isSubscribedCivicLiker,
     civicLikerSince,
+    description: description || '',
```

Comment:
can we just do `description`?
```suggestion
    description,
```